docs/patch.md

```diff
@@ -220,7 +161,7 @@ The `combine` method has the following signature:
 patch.combine(path: string, value: JsonArray | JsonMap): this
```

Comment:
`patch.merge`

Reply:
And above:

> The `merge` method has the following signature:

Reply:
That's weird, I remember that we changed these typos a while ago :(

Reply:
Anyways, here I go